kong/clustering/services/sync/rpc.lua

```diff
@@ -54,13 +54,30 @@ local function full_sync_result()
 end
 
 
+local function get_current_version()
+  return declarative.get_current_hash() or DECLARATIVE_EMPTY_CONFIG_HASH
+end
+
+
+local is_invalid_version
+do
+  local byte = string.byte
+  local CHAR_V = byte("V")
+
+  -- version string must start with char 'V'
```

Comment:
I feel this format is not good for future extensibility. Maybe:

`02xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`

Where `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx` is the hex encoded version number. Essentially: `"02%030x"`. This allows us to somewhat distinguish in the future when a new format is used. We can simply add `03`, `04`, etc.

Reply:
If the string is like `02xxx` then we can not distinguish between a hex string (ngx.md5) and a version string.

e.g.: a v1 config hash just starts with `02` (`02ABCDEF...`).

Reply:
Perhaps we could set the version string a `V0xxx` `V1xxx`

Reply:
@chronolaw The version string is provided by the DP, and DP should know if it is a valid version or not. If the string is not valid, then DP should simply send `02000000000000000000000000000000` instead. Does CP need to know about this detail?

Reply:
As I said above, if the version string stored in lmdb is `02ABCDEF...`, and it is a config hash generated by a YAML by change, how DP could judge it valid?

Reply:
changed to `V02_`